### PR TITLE
Update a position on item tap

### PR DESCRIPTION
**Pull request by badkos**
Calling `onChange` when the item was tapped.

### CODE REVIEW

**Comment by vdor** [comment]
@badkos - Thank you!

Can you update the code to this?
`
if (this.props.onChange != null) {
      this.props.onChange(position);
}
`

Also, please remove the call of `onChange` in the function `setDelayedSnap`. (line 141) and from the line 134.

File: src/index.tsx [hunk 1]
```diff
@@ -117,6 +117,10 @@ export default class HorizontalPicker extends PureComponent<HorizontalPickerProp
     if (this.refScrollView.current != null) {
       this.refScrollView.current.scrollTo({ x, y: 0, animated: true });
     }
+    
+    if (onChange != null) {
```
Comment (vdor) [inline src/index.tsx:121]: `this.props.onChange`

File: src/index.tsx [hunk 2]
```diff
@@ -117,6 +117,10 @@ export default class HorizontalPicker extends PureComponent<HorizontalPickerProp
...
+      onChange(position);
```
Comment (vdor) [inline src/index.tsx:122]: this.props.onChange

**Review by vdor — CHANGES_REQUESTED** [review]

**Comment by badkos** [comment]
@vdor  I apologize for the inattention

**Pull request merged**